easy/13.js

```diff
@@ -1,9 +1,9 @@
 function isPrefix(word, prefix) {
-	
+    return word.includes(prefix.substring(0, prefix.length - 1));

```

Comment:
test(isPrefix('unboxing', 'box-'), ???)

what should this example return? I think false, because this is not prefix. The same for suffix